DigitalLearningSolutions.Web.AutomatedUiTests/SeleniumServerFactory.cs

```diff
@@ -38,11 +36,22 @@ protected sealed override TestServer CreateServer(IWebHostBuilder builder)
 
         protected sealed override IWebHostBuilder CreateWebHostBuilder()
         {
-            var builder = WebHost.CreateDefaultBuilder(Array.Empty<string>());
-            builder.UseStartup<TStartup>();
-            builder.UseSerilog();
-            ConfigureWebHost(builder);
-            return builder;
+            return WebHost.CreateDefaultBuilder(Array.Empty<string>())
+                .UseStartup<TStartup>()
+                .UseSerilog()
+                .ConfigureAppConfiguration(configBuilder =>
+                {
+                    var jsonConfigSources = configBuilder.Sources
+                        .Where(source => source.GetType() == typeof(JsonConfigurationSource))
+                        .ToList();
+
+                    foreach (var jsonConfigSource in jsonConfigSources)
+                    {
+                        configBuilder.Sources.Remove(jsonConfigSource);
```

Comment:
I take it this is removing config information to be replaced with settings from the SIT.json file. Does it not automatically override? Or have I misunderstood what's going on here?

This shouldn't block merging if its works, either way.

Reply:
Yep - this is what is happening. I created a new config for the app we're running in the server during SITs, where default db connection string is the test db connection string. This resolves the Jenkins issues, but also ensures that developers don't accidentally use the normal db connection for SITs. Hopefully having the SIT config separate makes it really clear how the SITs are configured in case we have issues in the future.